features/bundle/package.json

```diff
@@ -59,7 +65,9 @@
     {
       "id": "tools",
       "name": "Tools",
-      "features": []
+      "features": [
+        "test"
```

Comment:
This is where I would now place _demo_ as well as _test_, but it means we need to manually remove both before we make a release. I'll create a followup ticket to make this easier to deal with.